modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -70,6 +74,9 @@ public class PartitionListener implements RaftGroupListener {
     /** Logger. */
     private static final IgniteLogger LOG = Loggers.forClass(PartitionListener.class);
 
+    /** Empty sorted set. */
+    private static final SortedSet<RowId> EMPTY_SET = new TreeSet<>();
```

Comment:
Let's wrap it in `unmodifiableSortedSet()`, to make sure noone adds anything. This should not hit us hard performance-wise.

Reply:
I couldn't find such thing for some reason, maybe I didn't look hard enough. Good idea, I'll do it